modules/ROOT/pages/6.3-release-notes.adoc

```diff
@@ -0,0 +1,694 @@
+
+In {productname} 6.3, `getCurrentColor` now checks for the first found `backcolor` and `forecolor` color before the parent color, based on the current selection or insertion point position.
+
+As a consequence, when either the Text color or Background color toolbar menus are opened, if the selected Text color or Background color matches one of the colors available from the appropriate color swatch menu, that color is pre-selected and navigating through the available colors using arrow keys on the keyboard starts from this pre-selected color.
+
+=== `setContent` is now allowed to accept any custom keys and values as a second options argument
+
+In {productname} 6.3, changes have been made to the `setContent` feature that now allows the editor to accept any custom `[key: string]: (value) any;` as a second option argument when the user inserts content.
+
+The provides more flexibility with the `getContent` and `setContent` features.
+
+New tests have been added to the `setContent` feature in {productname} 6.3 that now check if content manipulation in `BeforeSetContent` callback is possible so it can pass the custom data object to the `getContent` feature, allowing it to set both the content and action events.
+
+
+[[bug-fixes]]
+== Bug fixes
+
+{productname} 6.3 also includes the following bug fixes:
+
+=== Parsing media content would cause a memory leak, which for example occurred when using the getContent API
+
+A memory leak occurred in Chrome, Edge and Firefox browsers when media content was parsed into the editor using the `tinymce.activeEditor.getContent()` method API.
+
+The memory leak caused an abnormal behavior that increased the memory requirements of the editor until the active tab in the browser crashed.
+
+This memory leak has been reported as https://bugs.chromium.org/p/chromium/issues/detail?id=1374180[Memory leak with DOMParser parseFromString method] in the Chromium bug tracker.
+
+A fix that prevents {productname} being affected by the leak has been included in {productname} 6.3.
+
+=== Dragging a noneditable element toward the bottom edge would cause the page to scroll up
+
+A regression was found in {productname} 6.2 where dragging an element containing a `contentEditable=false` attribute towards a document’s edge did not cause scrolling.
+
+It was also found that dragging elements containing a `contentEditable=false` attribute toward the bottom edge causes similar behavior. In this instance, however, when the elements containing a `contentEditable=false` attribute was dragged to the bottom of the page the editor would sometimes scroll up.
+
+This regression has been addressed in {productname} 6.3. Dragging elements containing a `contentEditable=false` attribute towards a document’s edge causes the viewport to scroll as expected. And dragging elements containing a `contentEditable=false` attribute toward the bottom edge of the editor no longer causes the page to scroll up.
```

Comment:
```suggestion
This regression has been addressed in {productname} 6.3. Dragging elements containing a `contenteditable=false` attribute towards a document’s edge causes the viewport to scroll as expected. And dragging elements containing a `contenteditable=false` attribute toward the bottom edge of the editor no longer causes the page to scroll up.
```

Reply:
Even though the HTML attribute is case-insensitive, we should be consistent across our docs/demos/examples.